### DISCUSSION TITLE
Configurable delay on led color changes for gentle / smooth ambience.

### DISCUSSION THREAD

**Discussion by Teggles** [krojew/OpenRGB-Ambient#8] [Ideas]
Hey there, I'm absolutely loving your plugin and am so excited at the eventual prospect of being able to replace my 4 different RGB apps with a single one. It's already replaced NZXT Cam for me.

One feature that would really make it perfect for me would be if we could add a duration for a delay on the transition, so the room is less of an instant extension of the screen and more of a gentle glow representing the color vibe of what's been happening.

For reference, the Philips Hue sync system supports "subtle", "moderate", "high" and "extreme" and this feels like it's locked at extreme, but I usually used subtle.

**Comment by krojew** [comment]
Sounds like a good idea. Could be configurable via a slider.

**Comment by krojew** [comment]
You can try the latest release (2.3.0) for transition smoothing. Note: when changing smoothing settings in the Ambient tab, go to any other tab to apply the changes.

**Comment by Teggles** [comment]
AMAZING, it works perfect and was exactly what I wanted. You're fantastic.

(one note, there's no visual indication of which direction is "more subtle" so I instinctually moved it to the right to make it smoother, but it was actually to the left. Perhaps add + and - icons on your next version?)

**Reply by krojew** [reply]
Good point. I forgot about proper labeling.
